wgsl/index.bs

```diff
@@ -1753,6 +1753,15 @@ The originating variable for a formal parameter of a function depends on the
 [=call site|call sites=] for the function.
 Different call sites may supply pointers into different originating variables.
 
+If a reference or pointer access is out of bounds, an <dfn noexport>invalid
+memory reference</dfn> is produced.
+[=Load Rule|Loads=] from an invalid reference return either:
+    * a value from any [=memory locations|memory location=] of the [=originating variable=]
+    * the zero value for store type of the reference
```

Comment:
Vulkan's RBA also defines a few other cases:
> 

Zero values, or (0,0,0,x) vectors for vector reads where x is a valid value represented in the type of the vector components and may be any of:
>    0, 1, or the maximum representable positive integer value, for signed or unsigned integer components
>    0.0 or 1.0, for floating-point components

I don't know if it makes sense to define these, or just throw it all up and say the returned value can be anything.

Reply:
What about the last part, which I'm not seeing covered:
> (0,0,0,x) vectors for vector reads where x is a valid value represented in the type of the vector components and may be any of:
    0, 1, or the maximum representable positive integer value, for signed or unsigned integer components
    0.0 or 1.0, for floating-point components

Reply:
Done in the latest commit (which also squashed and rebased).